bindings.c

```diff
@@ -584,19 +584,26 @@ static bool write_string(const char *fnam, const char *string, int fd)
 	FILE *f;
 	size_t len, ret;
 
-	if (!(f = fdopen(fd, "w")))
+	f = fdopen(fd, "w");
+	if (!f)
 		return false;
+
 	len = strlen(string);
 	ret = fwrite(string, 1, len, f);
 	if (ret != len) {
-		lxcfs_error("Error writing to file: %s\n", strerror(errno));
+		lxcfs_error("%s - Error writing \"%s\" to \"%s\"\n",
+			    strerror(errno), string ? string : "(null)",
```

Comment:
I don't think `string` can be `NULL` here? Otherwise the `strlen(string)` and `fwrite(string, ...)` would already be an issue.

Reply:
Thanks for spotting.

Reply:
Fixed.